parts/xmem_nozzle.go

```diff
@@ -869,7 +890,16 @@ func (xmem *XmemNozzle) Start(settings metadata.ReplicationSettingsMap) error {
 		return err
 	}
 
-	xmem.Logger().Infof("%v has been started", xmem.Id())
+	if xmem.source_cr_mode == base.CRMode_Custom || xmem.config.crossClusterVers {
+		xmem.dataPool = base.NewDataPool()
+	}
+
+	if xmem.config.crossClusterVers {
+		xmem.Logger().Infof("%v has been started with %v=%v, %v=%v", xmem.Id(),
```

Comment:
Remove customized message. Keep line 901 as original message.

All the settings should have been printed as part of XMEM start
```
	xmem.Logger().Infof("%v starting ....settings=%v\n", xmem.Id(), settings.CloneAndRedact())
```
You can add the pulled settings into the settings map to be printed

Reply:
Done